src/components/Schedule.js

```diff
@@ -5,62 +5,95 @@ import Arrow from "../assets/arrow.svg";
 import data from "../data/fake_data.json";
 
 const Schedule = () => {
```

Comment:
hmm maybe we need some pre commit hook? I think my format ran this file again ;-;